### DISCUSSION TITLE
expose-loader objects are not immediately available because of defer="defer" ?

### DISCUSSION THREAD

**Discussion by patrick99e99** [webpack/webpack#14741] [General]
my html document's head tag has in it:
  `<script src="/packs/vendors-node_modules_jquery_dist_jquery_js-exposed-e9b29cc0e77d15c93346.js" defer="defer"></script>`
  
and in the document's body, I have a script tag which is trying to use jQuery, but it says `$ is undefined`.

How can I make it so that expose-loader does not put that defer tag on jQuery so that it's available when my script tag is executed?

**Comment by alexander-akait** [comment]
Put `jQuery` above your script

**Comment by patrick99e99** [comment]
jQuery is above my script... I am saying my html document is like this:

```
  ...
  <head>
     <script src="/packs/vendors-node_modules_jquery_dist_jquery_js-exposed-e9b29cc0e77d15c93346.js" defer="defer"></script>
  </head>
  <body>
     ...
     <script>
       $.doSomething();
     </script>
     ...
```

$ is undefined because the `defer="defer"` tag.  How do I tell webpack to not put that defer tag on the jquery script tag?
